src/firefox/remote.js

```diff
@@ -4,32 +4,58 @@ import {RemoteTempInstallNotSupported, WebExtError} from '../errors';
 import defaultFirefoxConnector from 'node-firefox-connect';
 
 const log = createLogger(__filename);
+
+
+// Flow Types
+
+import type FirefoxClient from 'firefox-client';
+
+export type FirefoxConnectorFn =
+  (port?: number) => Promise<FirefoxClient>;
+
+export type ConnectOptions = {
```

Comment:
I would prefer to see this above the [connect()](https://github.com/rpl/web-ext/blob/d5083f42002b2dcb5e72f8c886997d0035c1c0c7/src/firefox/remote.js#L196) function